pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 5.0.0
+2021-07-26 
+
+* Replaced STAR alignment with STARsolo and modified the structure of the wdl to utilize the umi and barcode correction from STARsolo. In the new implementation of Optimus STARsolo takes the fastq file and directly creates a count matrix file and a BAM file.
```

Comment:
Since this is a Major release, let's be a little more thorough in our changelog notes. Typically a major release contains breaking changes, so please note any changes (or lack thereof) to inputs and outputs. Also note whether the results of this version will be comparable to previous versions.

Reply:
```suggestion
* Replaced STAR alignment with STARsolo and modified the structure of the WDL to utilize the UMI and barcode correction from STARsolo. In the new implementation of Optimus, STARsolo uses the FASTQ file as input and directly creates a count matrix file and a BAM file. No updates have been made to the inputs or outputs. The outputs for this version are identical to the outputs for the previous Optimus version.

Reply:
Please commit the suggestion tha Liz provided ^